static/app/components/searchQueryBuilder/tokens/filter/valueListBox.tsx

```diff
@@ -12,11 +12,48 @@ import {itemIsSection} from 'sentry/components/searchQueryBuilder/tokens/utils';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
 
+interface ConstrainAndAlignListBoxArgs {
+  popoverRef: React.RefObject<HTMLElement | null>;
+  referenceRef: React.RefObject<HTMLElement | null>;
+  refsToSync: Array<React.RefObject<HTMLElement | null>>;
+}
+
+export function constrainAndAlignListBox({
```

Comment:
This approach feels a bit manual and not very “react-y”. Can you maybe show the steps to reproduce for the original issue? I’ve tried out the search bar and I don’t see it stretching like shown in the original screenshots on the pages I’ve tried 🤔